composer.json

```diff
@@ -27,7 +27,6 @@
         "yiisoft/aliases": "^1.1|^2.0|^3.0",
         "yiisoft/db": "^1.1",
         "yiisoft/di": "^1.0",
-        "yiisoft/definitions": "^3.0",
```

Comment:
This package used in [MigrationContainer](https://github.com/yiisoft/db-migration/blob/master/bin/MigrationContainer.php)

Reply:
I closed the PR

But it looks they can be replaced with

```php
[
    'migrate:create' => CreateCommand::class,
    'migrate:down' => DownCommand::class,
    'migrate:history' => HistoryCommand::class,
    'migrate:new' => NewCommand::class,
    'migrate:redo' => RedoCommand::class,
    'migrate:up' => UpdateCommand::class,
]
```